### ISSUE TITLE
Author affiliation missing in ScopusAbstract

### ISSUE THREAD

**Issue by varputa** [scopus-api/scopus#53]
Hi, I am using ScopusAbstract and have a problem. 

```
from scopus import ScopusAbstract
ab = ScopusAbstract("2-s2.0-84930616647")
for au in ab.authors:
        print(au)
```

I can't get the author's affiliation id now, which is different from your example. My question is how I can get this information. I mean the author's affiliation when the paper got published, rather than one's current affiliation. 

Thanks a lot!

**Comment by Michael-E-Rose** [comment]
Hey varputa,

sorry for coming back to you so late.  There was a litte bug related to the namespace of the author affiliations in an abstrct which I had to fix and test first.
You can get the affiliation IDs listed on a publication as described in the [(updated) example page](https://scopus.readthedocs.io/en/latest/examples/ScopusAbstract.html):
```
>>> for aff in ab.affiliations:
...     print(aff.id)
...
60027950

```
I just relased version 0.5, so you might want to upgrade first.

And now that I write I realize that the affiliation is really not part of the scopus author information.  Will fix this soon as well.

**Comment by Michael-E-Rose** [comment]
Fixed in 94eeb4fd4c3a717b5146ba69ee71a0a9a1b2c0f8, will go live with the next release.
